spec/index.html

```diff
@@ -418,7 +418,10 @@ <h2>Simple Interpretations</h2>
               set of sets of pairs &lt; x, y &gt; with x and y in IR .</p>
         <p>4. A mapping IS from IRIs into (IR union IP)</p>
         <p>5. A partial mapping IL from literals into IR </p>
-        <p>6. An injective mapping RE from IR x IP x IR into IR, called the denotation of triple terms. </p>
+        <p data-tests="rdf-semantics/index.html#all-identical-triple-terms-are-the-same,
```

Comment:
Every element with `data-tests` needs to have an anchor, so that it can be back-referenced from the manifest. Otherwise, looks good.

```suggestion
        <p id="simple-injective-tests" data-tests="rdf-semantics/index.html#all-identical-triple-terms-are-the-same,
```